text/0060-tasks-api.md

```diff
@@ -713,7 +713,7 @@ The tasks API endpoints are filterable by  `uids`, `indexUids`, `types`, `status
 
 `uids` is **case-unsensitive**.
 
-- 🔴 Sending values with a different type than `Integer` being separated by `,` for the `uid` parameter returns an [`invalid_task_uids_filter`](0061-error-format-and-definitions.md#invalid_task_uids_filter) error.
+- 🔴 Sending values with a different type than `Integer` being separated by `,` for the `uid` parameter returns an [`invalid_task_uids`](0061-error-format-and-definitions.md#invalid_task_uids) error.
 
 ###### 11.2.2. `indexUids`
```

Comment:
No error are specified here, but we can trigger one `invalid_index_uid`

Reply:
Good catch, now we have deserr we can easily return an `invalid_task_index_uids` if one of the provided `index` uids is invalid regarding our expected format!

Reply:
📡 Just specified `invalid_task_index_uids`